terraform/modules/lambda/iam.tf

```diff
@@ -1,5 +1,7 @@
 resource "aws_iam_role" "lambda" {
-  name               = "lambda-${var.lambda_name}-${var.environment_name}-${data.aws_region.current.name}"
+  name = "lambda-${var.lambda_name}-${var.environment_name}-${data.aws_region.current.name}"
+  # this path will be used to grant permission to the lambda to access the KMS key
+  # path               = "/lpa-store-lambda/"
```

Comment:
is this needed?

Reply:
it's useful for the PR that follows this one